packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import AsyncCreatableSelectField from './async-creatable-select-field';
+import { useEffect, useState } from 'react';
+import { iconArgType } from '@/storybook-helpers';
+
+const meta: Meta<typeof AsyncCreatableSelectField> = {
+  title: 'Form/Fields/AsyncCreatableSelectField',
+  // @ts-expect-error, @todo: fix typings of field component
+  component: AsyncCreatableSelectField,
+  argTypes: {
```

Comment:
It's odd that the `infoButton` is being displayed in the example since there isn't anything being explicitly passed to `onInfoButtonClick`, and the control isn't editable.
![Screenshot 2024-08-15 at 3 36 52 PM](https://github.com/user-attachments/assets/687a04b9-f029-45c1-b406-36f289ade0de)
![Screenshot 2024-08-15 at 3 36 58 PM](https://github.com/user-attachments/assets/caa30a28-b418-4a94-95f4-e88bb01c04ab)

Reply:
Storybook adds some sort of handler, I suppose for this:

![image](https://github.com/user-attachments/assets/085a28a3-91f0-45b5-b42c-b134dbca0067)